packages/kokoas-client/src/pages/projInvoice/formValidation.ts

```diff
@@ -11,6 +11,22 @@ const numberValidation = Yup
   .number()
   .typeError('数字を入力してください');
 
+
+/* unique validations */
+const billingAmountValidation = function (this: any) {
```

Comment:
```suggestion
const billingAmountValidation = function (this: { parent: TypeOfForm['estimates'][number] }) {
```

変更に簡単に対応出来るように、`any` は出来るだけ使わないほうがいいと言われています。

Reply:
ご指摘ありがとうございます。
[```b44090c```](https://github.com/Lorenzras/yumecoco-monorepo/pull/133/commits/b44090ca94b61ecc4366280904fdcda89ffece54)にて修正しました。